prow/config.yaml

```diff
@@ -155,6 +155,28 @@ branch-protection:
               protect: true
             release-1.9:
               protect: true
+    kubernetes-csi:
+      protect: true
+      required_status_checks:
+        contexts:
+        - cla/linuxfoundation
+      repos:
+        csi-test:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
+        drivers:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
+        external-attacher:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
+        external-provisioner:
+          required_status_checks:
+            contexts:
+            - continuous-integration/travis-ci
```

Comment:
Travis CI is also enabled in these repos:
* https://github.com/kubernetes-csi/driver-registrar
* https://github.com/kubernetes-csi/external-snapshotter
* https://github.com/kubernetes-csi/livenessprobe

Not sure if they should also be specified here.

Reply:
When I was looking at the repo settings, those contexts weren't yet listed as required for merge.  I'd rather keep things exactly as they are, and then step away and let you make any further changes you'd like via PR.

Reply:
Makes sense. SGTM!